src/components/organisms/LinkCard.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import PropType from 'prop-types';
+import './LinkCard.scss';
+
+const LinkCard = ({ story }) => (
+  <article className="card">
+    <div className="side-bar">
+      <div className="side-bar-icon">icon</div>
+      <div className="side-bar-links">
+        <p>links</p>
+        <p>stars</p>
+      </div>
+    </div>
+    <div className="card-info">
+      <header>
+        <span className="byline">
+          {`${story.authorName} on ${new Date(story.createdDate * 1000).toLocaleDateString('en-US', { year: 'numeric', month: 'long', day: '2-digit' })}`}
+        </span>
+        <h3>{story.title}</h3>
+      </header>
+      <p>
+        {story.content}
+      </p>
+    </div>
+  </article>
+);
+
+export default LinkCard;
+
+LinkCard.propTypes = {
+  story: PropType.arrayOf(Object).isRequired,
```

Comment:
here you're saying you're expecting the `story` prop to come in as an array of objects. however it seems like story should just be an object, no? you're also getting an error on the console about this very thing.

Reply:
to be totally honest, i just copied this from StoryList.js and am not really sure what it's purpose is.